Client/Frontend/Browser/Playlist/Managers & Cache/PlaylistCacheLoader.swift

```diff
@@ -456,13 +456,21 @@ class PlaylistWebLoader: UIView {
       self.webLoader = webLoader
     }
 
-    static func name() -> String {
-      return "PlaylistWebLoader"
-    }
-
-    func scriptMessageHandlerName() -> String? {
-      return "playlistCacheLoader_\(UserScriptManager.messageHandlerTokenString)"
-    }
+    static let scriptName = "PlaylistWebLoaderContentHelper"
+    static let scriptId = UUID().uuidString
+    static let messageHandlerName = "\(scriptName)_\(messageUUID)"
+    static let scriptSandbox: WKContentWorld = .page
+    static let userScript: WKUserScript? = {
+      guard var script = loadUserScript(named: scriptName) else {
+        return nil
+      }
+      return WKUserScript.create(source: secureScript(handlerName: messageHandlerName,
+                                                      securityToken: scriptId,
+                                                      script: script),
+                                 injectionTime: .atDocumentStart,
+                                 forMainFrameOnly: false,
+                                 in: scriptSandbox)
+    }()
 
     func userContentController(_ userContentController: WKUserContentController, didReceiveScriptMessage message: WKScriptMessage, replyHandler: (Any?, String?) -> Void) {
```

Comment:
Here and a few other handlers don't check for security tokens. As discussed in the VPN SKU's PR I don't think it's always a security issue, especially when we inject into a separate content world like in this example. Should we scrap security tokens for sandboxed scripts?

Reply:
Actually, this handler isn't sandboxed :( Should definitely add a security token check here. But the question still stands for the sanboxed scripts

Reply:
Fixed.